src/ds/d400/d400-device.h

```diff
@@ -65,16 +65,17 @@ namespace librealsense
         bool check_fw_compatibility(const std::vector<uint8_t>& image) const override;
     protected:
         std::shared_ptr<ds_device_common> _ds_device_common;
-        virtual void register_interleaved_y16_processing_block(synthetic_sensor& depth_sensor) const;
 
         std::vector<uint8_t> get_d400_raw_calibration_table(ds::d400_calibration_table_id table_id) const;
-        virtual ds::d400_caps parse_device_capabilities() const;
+        ds::ds_caps parse_device_capabilities() const;
```

Comment:
please check this method is needed, considering you have added the same method that gets buffer as input

Reply:
Removed, good catch